tasks/asserts.yml

```diff
@@ -40,3 +40,11 @@
   debug:
     msg: "vault_redirect_address is deprecated. Check for vault_api_addr in the README."
   when: vault_redirect_address is defined
+
+- name: Check if vault_transit_address and vault_transit_token has been specified
+  fail:
+    msg: "need vault_transit_address and vault_transit_token defined for vault transit seal configuration."
+  when:
+    - vault_transit | bool
+    - not vault_transit_address
+    - not vault_transit_token
```

Comment:
`when` items are combined with an `and` operator. These two lines should be grouped together and use an `or` operator to trigger the `fail` task when either one is missing.